lms/djangoapps/courseware/module_render.py

```diff
@@ -475,13 +476,15 @@ def publish(block, event_type, event):
                 only_if_higher=event.get('only_if_higher'),
             )
         else:
-            aside_context = {}
+            context = contexts.course_context_from_course_id(course_id)
+            context['user_id'] = block.runtime.user_id
+            context['asides'] = {}
             for aside in block.runtime.get_asides(block):
                 if hasattr(aside, 'get_event_context'):
                     aside_event_info = aside.get_event_context(event_type, event)
                     if aside_event_info is not None:
-                        aside_context[aside.scope_ids.block_type] = aside_event_info
-            with tracker.get_tracker().context('asides', {'asides': aside_context}):
+                        context['asides'][aside.scope_ids.block_type] = aside_event_info
+            with tracker.get_tracker().context(event_type, context):
```

Comment:
As far as I was able to tell, the context on the event ends up looking the same either way, just with the addition of the course and user id's. The first parameter just seems to be a name for the context that doesn't appear on the event itself. Since the context here now includes more than just asides information, I changed the name to be in keeping with the general practice of naming the context for the event type.

@brianhw did I get that right?